src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  let totalCost = days * dailyCost;
```

Comment:
Prefer `const` over `let` for `totalCost` since it's not being reassigned, but only being modified within the conditional blocks.